modules/oadp-configuring-velero-plugins.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: PROCEDIRE
+[id="oadp-configuring-velero-plugins_{context}"]
+= Configuring OADP Velero plug-ins
+
+You can configure two types of plug-ins when you install Velero:
+
+* Default cloud provider plug-ins
+* Custom plug-ins
+
+Both types of plug-in are optional, but most users configure at least one cloud provider plug-in.
+
+== Default Velero cloud provider plug-ins
+
+You can install any of the following types of default Velero cloud provider plug-ins when you configure the `oadp_v1alpha1_dpa.yaml` file during deployment:
+
+* `aws` (Amazon Web Services)
+* `gcp` (Google Cloud Platform)
+* `azure` (Microsoft Azure)
+* `openshift` (OpenShift Velero Plugin)
+* `csi` (Container Storage Interface)
+* `kubevirt` (KubeVirt)
+
+== Custom Velero plug-ins
+
+You can install a custom Velero plug-in by specifying the plug-in `image` and `name` when you configure the `oadp_v1alpha1_dpa.yaml` file during deployment.
+
+.Procedure
```

Comment:
I'm not sure all of this is suppose to be on a "procedure".
I think it's better to just specify the different plugin types as options, like in the upstream doc:
https://github.com/openshift/oadp-operator/blob/master/docs/config/plugins.md

It may cause a confusion and the steps make it look like custom plguin is dependent on default plugins existence...

Reply:
IMO, yes

Reply:
OK